scripts/run-api-tests.sh

```diff
@@ -1,16 +1,6 @@
 #!/bin/sh
 
-script="$(dirname $0)/../docker-compose-api-tests.yml"
-docker compose -p tests -f $script up --build -d
-
-TESTS_RUNNER_CONTAINER_ID=$(docker ps -aqf "name=^tests-api-tests-runner-1$")
-TEST_EXIT_CODE=`docker wait $TESTS_RUNNER_CONTAINER_ID`
-
-# Log tests results
-docker logs $TESTS_RUNNER_CONTAINER_ID
-
-# Clean
-docker-compose -p tests kill
-docker-compose -p tests rm -f
-
-exit $TEST_EXIT_CODE
+TEST_COMPOSE="$(dirname "$0")/../docker-compose-api-tests.yml"
+docker compose -p tests -f "$TEST_COMPOSE" build
+docker compose -p tests -f "$TEST_COMPOSE" run api-tests-runner
+exit $?
```

Comment:
returns with the same output & same error code

Reply:
cool... works